cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,14 @@
+class ProductsListPage{
+
+    private productsList: string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with?